dpsim-models/src/SimNode.cpp

```diff
@@ -39,10 +39,10 @@ SimNode<VarType>::SimNode(PhaseType phaseType)
 
 template <>
 void SimNode<Real>::initialize() {
-	(**mVoltage)(0,0) = (**mInitialVoltage)(0,0).real();
-	if (phaseType() == PhaseType::ABC) {
-		(**mVoltage)(1,0) = (**mInitialVoltage)(1,0).real();
-		(**mVoltage)(2,0) = (**mInitialVoltage)(2,0).real();
+	if (phaseType() == PhaseType::Single) {
+		(**mVoltage)(0,0) = (**mInitialVoltage)(0,0).real();
```

Comment:
Doesn't this need also the multiplication of `mInitialVoltage` with `RMS3PH_TO_PEAK1PH`?

Reply:
This has been changed accordingly. The [guideline for the scaling of voltages and currents](https://github.com/sogno-platform/dpsim/blob/4a79f7f95c4149935b0fa1d5f6a3c6050b1b342f/docs/hugo/content/en/docs/Development/Guidelines.md) is now documented.